sts-spring/src/main/java/de/adorsys/sts/token/tokenexchange/server/TokenExchangeRestController.java

```diff
@@ -12,4 +13,8 @@
 public class TokenExchangeRestController extends TokenExchangeController {
```

Comment:
We need no path variable. Make like in the other classes 
@RequestMapping("${sts.secret-server.endpoint:/secret-server/token-exchange}")